samples/v1/types_config.go

```diff
@@ -64,6 +64,15 @@ type ConfigSpec struct {
 	// content but the operator will not recreate(or update) anything
 	// listed here.
 	SkippedTemplates []string `json:"skippedTemplates,omitempty" protobuf:"bytes,6,opt,name=skippedTemplates"`
+
+	// skippedhelmcharts specifies names of helm charts that should NOT be
+	// managed. Admins can use this to allow them to delete content
+	// they don’t want. They will still have to MANUALLY DELETE the
+	// content but the operator will not recreate(or update) anything
+	// listed here.
+	// +listType=set
+	// +kubebuilder:validation:MaxItems=16
+	SkippedHelmCharts []string `json:"skippedHelmCharts,omitempty" protobuf:"bytes,7,opt,name=skippedhelmCharts"`
```

Comment:
Where would I find the names of the helm charts?

Is there a format for the names of the helm charts? I assume they must fit the Kube standard [subdomain name](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names) right?

Reply:
The list of helmcharts is still being processed which are going to be available by default. and then the customer can use it skippedHelmcharts to skip them .The maximum would be 12, but they would come as the the imagestream owners convert them to helmcharts(by ocp4.17). once that is available we plan on making that list and put it to the product release notes for samples operator available to the customer. 
Also the customer would be able to see the list in openshift shift console in the  samples . 
This is as a part of Downsizing effort of CLuster Samples Operator and moving away from imagestreams.

Reply:
It would be good to capture some of this context within the godoc. So for example, "To find available helm charts, check the openshift console ..." would be helpful context for users

Reply:
okay updated it in the new PR